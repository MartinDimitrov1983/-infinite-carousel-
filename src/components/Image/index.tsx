import React from 'react';
import styles from './index.module.css';

interface ImageProps {
  src: string;
  alt: string;
  isActive: boolean;
}
const Image: React.FC<ImageProps> = ({ src, alt, isActive }) => (
  <img
    className={styles.img}
    src={src}
    alt={alt}
    style={{
      display: `${isActive ? 'inline-block' : 'none'}`,
    }}
  />
);

export default Image;
