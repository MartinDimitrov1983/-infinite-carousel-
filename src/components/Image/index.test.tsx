import React from 'react';
import { render } from '@testing-library/react';

import Image from './index';

test('Snapshot Input', () => {
  const { asFragment } = render(<Image src="" alt="" isActive={true} />);
  expect(asFragment()).toMatchSnapshot();
});
