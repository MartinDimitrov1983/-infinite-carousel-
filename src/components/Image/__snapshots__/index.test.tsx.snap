// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot Input 1`] = `
<DocumentFragment>
  <img
    alt=""
    class="img"
    src=""
    style="display: inline-block;"
  />
</DocumentFragment>
`;
